x-pack/qa/smoke-test-plugins-ssl/src/test/java/org/elasticsearch/smoketest/SmokeTestMonitoringWithSecurityIT.java

```diff
@@ -68,35 +61,6 @@ public class TestRestHighLevelClient extends RestHighLevelClient {
         }
     }
 
-    /**
-     * A JUnit class level rule that runs after the AfterClass method in {@link ESIntegTestCase},
-     * which stops the cluster. After the cluster is stopped, there are a few netty threads that
-     * can linger, so we wait for them to finish otherwise these lingering threads can intermittently
-     * trigger the thread leak detector
-     */
-    @ClassRule
-    public static final ExternalResource STOP_NETTY_RESOURCE = new ExternalResource() {
```

Comment:
This is an `ESRestTestCase` which doesn't derive from `ESIntegTestCase` so I think we are missing another call to `awaitGlobalNettyThreadsFinish` here.

Reply:
Ah well spotted, didn't even notice that to be honest. Though I think this check is needless here, we don't spin up any Netty things in the rest tests so this wasn't doing anything and it's fine that it goes away here I think.

Reply:
Ah yes that makes sense we don't start any nodes ourselves in these tests.